.gitignore

```
@@ -1 +1,2 @@
 *.obj
+*.out
```

please consider the following
```suggestion
*.obj
*.out
*.o
*.exe
```

---

OK. I'll do it.